packages/plans-grid/src/plans-table/plan-item.tsx

```diff
@@ -114,16 +136,34 @@ const PlanItem: React.FunctionComponent< Props > = ( {
 						</div>
 
 						<div className="plan-item__actions">
-							<Button
-								className="plan-item__select-button"
-								onClick={ () => {
-									onSelect( slug );
-								} }
-								isPrimary
-								disabled={ !! disabledLabel }
-							>
-								<span>{ __( 'Choose', __i18n_text_domain__ ) }</span>
-							</Button>
+							{ CTAVariation === 'NORMAL' ? (
```

Comment:
If we export the `CTAVariation` values, we can use them here instead of hard coding `'NORMAL'`

Reply:
I think exporting the type is enough. The IDE won't let you set an invalid value.